### PR TITLE
Update elysia protocol

### PR DESCRIPTION
**Pull request by donguks**
**NOTE**

> If you would like to add a `volume` adapter please submit the PR [here](https://github.com/DefiLlama/adapters).

1. Sorry, We no longer accept fetch adapter for new projects, we prefer the tvl to computed from blockchain data, if you have trouble with creating a the adapter, please hop onto our discord, we are happy to assist you.
2. The protocol is usually listed within 24 hours of merging the PR
3. Please fill the form below  **only if the PR is for listing a new protocol** else it can be ignored/replaced with reason/details about the PR
4. **For updating listing info** It is a different repo, you can find your listing in this file: https://github.com/DefiLlama/defillama-server/blob/master/defi/src/protocols/data2.ts, you can  edit it there and put up a PR
5. Do not edit/push `package-lock.json` file as part of your changes, we use lockfileVersion 2, and most use v1 and using that messes up our CI
6. No need to go to our discord and announce that you've created a PR, we monitor all PRs and will review it asap

---
Hi team! I love DefiLlama.

I updated our protocol's adapter.
Previous adapter only added erc20 balances of our moneypool for TVL. That was wrong.
Our protocol's TVL is sum of the amount deposited.
This PR use our DataPipeline contract that return moneypool status like deposited amount and borrowed ammount.
I hope this PR will be applied soon.

`node test.js projects/elysia/index.js` result is blow.

```
--- ethereum-borrowed ---
DAI                       296.03 k
USDC                      173.66 k
USDT                      164.55 k
Total: 634.25 k 

--- ethereum-staking ---
EL                        872.89 k
ELFI                      109.46 k
Total: 982.35 k 

--- ethereum ---
DAI                       409.69 k
USDT                      306.74 k
USDC                      177.63 k
Total: 894.06 k 

--- bsc-borrowed ---
BUSD                      698.23 k
Total: 698.23 k 

--- bsc ---
BUSD                      770.37 k
Total: 770.37 k 

--- bsc-staking ---
ELFI                      52.39 k
Total: 52.39 k 

--- borrowed ---
BUSD                      698.23 k
DAI                       296.03 k
USDC                      173.66 k
USDT                      164.55 k
Total: 1.33 M 

--- staking ---
EL                        872.89 k
ELFI                      161.84 k
Total: 1.03 M 

--- tvl ---
BUSD                      770.37 k
DAI                       409.69 k
USDT                      306.74 k
USDC                      177.63 k
Total: 1.66 M 

------ TVL ------
ethereum-borrowed         634.25 k
ethereum-staking          982.35 k
ethereum                  894.06 k
bsc-borrowed              698.23 k
bsc                       770.37 k
bsc-staking               52.39 k
borrowed                  1.33 M
staking                   1.03 M

total                    1.66 M 
```

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/elysia exports TVL: 
        
 
 
    bsc-borrowed              698.23 k
    bsc                       770.37 k
    bsc-staking               52.39 k
    ethereum                  894.02 k
    ethereum-borrowed         634.21 k
    ethereum-staking          984.03 k
    borrowed                  1.33 M
    staking                   1.04 M
    
    total                    1.66 M

File: projects/elysia/index.js [hunk 1]
```diff
@@ -24,46 +25,82 @@ const addresses = {
...
+    return balances;
+  };
 }
 
 module.exports = {
   ethereum: {
-    borrowed: borrowed('ethereum'),
-    tvl: sumTokensExport({ tokensAndOwners: moneyPools.ethereum}),
+    borrowed: borrowed("ethereum"),
+    tvl: deposited("ethereum"),
```
Comment (g1nt0ki) [inline projects/elysia/index.js:95]: @donguks thanks for the PR but this includes the amount that is lent out right?

**Comment by g1nt0ki** [comment]
@donguks sorry, just read the PR description, for lending protocols, we count only the token amount in contracts not lent out as tvl

**Comment by donguks** [comment]
@g1nt0ki I didn't know that. I'm sorry.

**Pull request closed without merging**